modules/data.land/R/download_NEON_soildata.R

```diff
@@ -0,0 +1,85 @@
+##' @name download_NEON_soildata
+##' @description: 
+##' Download NEON Soil Water Content and Soil Salinity data by date and site name
+##' 
+##' @param site four letter NEON site code name(s). If no site is specified, it will download all of them (chr) (e.g "BART" or multiple c("SRER", "KONA", "BART"))
+##' @param avg averaging interval: either 30 minute OR 1 minute, default will return BOTH (chr) (e.g 1 or 30) 
```

Comment:
if the options are 1 and 30, why are they being passed as character as numeric? And when I look at the code that parses this arg, it's clear that you're making a numeric comparison to the numbers 1 and 30, not the strings "1" and "30". Also, you say 1 or 30 are the options but the default is "all" so clearly 1 and 30 are not the only options.